### ISSUE TITLE
Training Datasets

### ISSUE THREAD

**Issue by junjunjunjoon** [marcopesavento/Super-resolution-3D-Human-Shape-from-a-Single-Low-Resolution-Image#1]
Hello.

I plan to collect datasets to train new models and have difficulty creating training datasets such as HR and LR.

1) Is the second process of data set creation, the "mesh watertight" process required?
2) I want to know how to create datasets, how much data i need.
3) Can I make LR object and use by reducing the number of vortex in HR?
4) How long did it take to learn how many datasets in what hardware environment?
5) To add more, Could you release the pre-trained model and demo code?

**Comment by marcopesavento** [comment]
Hi,
1- Yes, the meshes must be watertight. If they are not, the methods to sample and label points to create the ground-truth will give incorrect values. I believe there are other packages (mesh_to_sdf i.e.) that perform better with non-watertight meshes but I recommend to make them watertight.
2- To create the dataset (RGB, Mask, camera parameters), we follow the same procedure as PiFU. Please, refer to their website for further details. As explained in the paper, we use 402 models from THuman2.0 for training and 124 for testing.
3- To create the LR objects, we apply the decimation filter of Meshlab. As shown in the paper, other approaches can be used but the results may be slightly different.
4- I am not sure what you are asking here. Do you mean training time? or the dataset creation time?
5- We released the pre-trained weights
